source/includes/api/recognition/_submissions.md

```diff
@@ -0,0 +1,113 @@
+## Submissions
+
+### Get all submissions
+
+An endpoint to get all submissions for a given campaign. Results are paginated and return 100 records per page.
+
+#### HTTP Request
+
+`GET /api/v2/campaigns/{campaign_id}/submissions`
+
+#### Query Parameters
+
+Parameter | Type | Description
+--------- | ---- | -----------
+page | `integer` | The page number that you wish to view. Optional
+status | `string` | Status of the submissions to be returned. Optional
+user_id | `integer` | ID of a sender that submissions were made by. Optional
+created_at_date | `string` | A parameter for scoping submissions that were created from that date onwards. Optional.
+created_at_end_date | `string` | A parameter for scoping submissions that were created on or before the given date. Optional.
```

Comment:
Should this be `date`, not `string`?

Reply:
Also what's the date format expected?

Reply:
I used `string` to keep it consistent with how we have it elsewhere in the docs (i.e. in self claims) but I can change it to date 👍
The expected format is ISO-8601. I'll add that in, thanks.